DigitalLearningSolutions.Data/Services/CourseContentService.cs

```diff
@@ -31,5 +33,43 @@ public CourseContentService(IDbConnection connection, ILogger<CourseContentServi
                 new { customisationId }
             );
         }
+
+        public int GetProgressId(int candidateId, int customisationId)
+        {
+            return connection.QueryFirstOrDefault<int>(
+                @"SELECT ProgressId
+                        FROM Progress
+                        WHERE CandidateId = @candidateId
+                          AND customisationId = @customisationId",

```

Comment:
I'm guessing this works since you've tested it, but I'd change it to match the database casing (uppercase ID, uppercase c in CustomisationID) just for consistency 🙂